src/main/java/gov/cabinetoffice/gapuserservice/util/WebUtil.java

```diff
@@ -36,6 +36,15 @@ public static Cookie buildSecureCookie(final String name, final String value) {
         return cookie;
     }
 
+    public static Cookie buildSecureCookie(final String name, final String domain, final String value) {
```

Comment:
probably best to swap the domain and value parameters so that the overloaded method call is more similar to the original:
```suggestion
    public static Cookie buildSecureCookie(final String name, final String value, final String domain) {
```

Reply:
Agreed, will change this in next push!